protostar-rust/tests/data/dispatchers/src/hello_starknet.cairo

```diff
@@ -23,14 +23,4 @@ mod HelloStarknet {
             self.balance.read()
         }
     }
-
-    #[external(v0)]
-    fn __validate_deploy__(
-        self: @ContractState,
-        class_hash: felt252,
-        contract_address_salt: felt252,
-        public_key_: felt252
-    ) -> felt252 {
-        starknet::VALIDATED
-    }
 }
```

Comment:
Why we need to remove that? cc @cptartur

Reply:
It's only required for making `DEPLOY_ACCOUNT` transactions, so not required now.